benchmark/benchmarks.jl

```diff
@@ -38,3 +38,16 @@ SUITE["with_quantity"] = let s = BenchmarkGroup()
     s["+y"] = @benchmarkable $f6(x, y) setup = (x = default(); y = x + rand() * x) evals = 1000
     s
 end
+
+SUITE["with_array"] = let s = BenchmarkGroup()
+    N = 10000
+    f9(x) = x^2
+    s["arr.^2"] = @benchmarkable $f9.(arr) setup = (arr = randn($N)) evals = 1000 # baseline
```

Comment:
The benchmarks on arr are just baselines that we wouldn't expect to change between branches -- I included them because it's useful to assess the overhead of quantity arrays, hopefully that's ok?

Reply:
Yes, good idea; sounds great!